docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. See the following table of contents to go to the devfile property you want to work with:
```

Comment:
Think I should say "users" here since it's not a procedure, so I can't directly address users? I want to make it user focus, but I don't want to assume the user is doing or thinking anything particular by saying 'you` here.

Reply:
I think using "you" is fine in here